### PR TITLE
COR-471: add tests for custom validation

### PR DESCRIPTION
**Pull request by internetti**
added tests for the custom validation functions

### CODE REVIEW

File: frontend/apps/pwa/src/features/former/__tests__/validation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import { FieldKind, FormType } from 'core-api-client';
+import { FieldPath } from 'react-hook-form';
+
+import { customValidation, validationConstants } from '../validation';
+import { ValidationForm } from '../../../reducers/Former/types';
+
+describe('validation', () => {
```
Comment (neb42) [inline frontend/apps/pwa/src/features/former/__tests__/validation.spec.ts:7]: I'm not sure the top level describe is necessary. The file handles this

**Comment by neb42** [comment]
Very minor comments. Happy to approve now, unless you want to make the changes.

File: frontend/apps/pwa/src/features/former/__tests__/validation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  describe('customValidation', () => {
+    describe('form', () => {
```
[the remark this answers is not in the document]
Comment (internetti) [reply frontend/apps/pwa/src/features/former/__tests__/validation.spec.ts:9]: @neb42 might as well do them now :+1:

**Pull request merged**